### PR TITLE
Jen nest

### PR DESCRIPTION
**Pull request by jenneft**
Hi sunshines! Please review my code at your earliest convenience. Thank you!

### CODE REVIEW

File: src/JenTasks/week3/t3_ReverseNegativeNumber.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
+package JenTasks.week3;
+
+public class t3_ReverseNegativeNumber {
```
[the remark this answers is not in the document]
Comment (jenneft) [reply src/JenTasks/week3/t3_ReverseNegativeNumber.java:3]: Yes, I misunderstood the task. I will reverse it in the am. Btw it also need to turn it into positive integer.

[the remark this answers is not in the document]
Comment (jenneft) [reply src/JenTasks/week3/t3_ReverseNegativeNumber.java:3]: excited output should be 1, 952, 1287, 2198

[the remark this answers is not in the document]
Comment (jenneft) [reply src/JenTasks/week3/t3_ReverseNegativeNumber.java:3]: wait so we don't need to change number into absolute number? only reverse the sequence then, am I correct?

File: src/JenTasks/week3/t1_PrimeNumber.java
Comment (cbanavi) [inline src/JenTasks/week3/t1_PrimeNumber.java:1]: lgtm

File: src/JenTasks/week3/t2_ArmstrongNumbers.java
Comment (cbanavi) [inline src/JenTasks/week3/t2_ArmstrongNumbers.java:1]: lgtm

File: src/JenTasks/week3/t3_ReverseNegativeNumber.java
Comment (cbanavi) [inline src/JenTasks/week3/t3_ReverseNegativeNumber.java:1]: lgtm

**Review by cbanavi — APPROVED** [review]
Good!

**Review by bernasantiago — COMMENTED** [review]
Lgtm!

**Pull request merged**